docs/modular-resources/advanced/create-subtype.md

```diff
@@ -4,76 +4,81 @@ linkTitle: "New API Subtype"
 
-{{% /alert %}}
+  {{% /alert %}}
 
 ## Define your new resource API
 
 Viam uses [protocol buffers](https://protobuf.dev/) for API definition.
 
-To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), write code in Python or Go to implement the higher level server and client functions required, and generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/).
+To define a new subtype, you need to define the methods and messages of the new API in [protobuf](https://github.com/protocolbuffers/protobuf), generate all necessary [protobuf module files](https://buf.build/docs/generate/usage/), and write code in Python or Go to implement the higher level server and client functions required.
 The following steps guide you through this process in more detail:
 
 1. Decide whether your subtype is a {{< glossary_tooltip term_id="component" text="component" >}} or a {{< glossary_tooltip term_id="service" text="service" >}}.
    If it provides an interface to control hardware, it is a component.
    If it provides higher-level functionality, it is a service.
 1. Choose a name for your subtype.
-   For example, "gizmo."
-1. Create a directory or repository for your module.
+   For example, "gizmo".
+1. Create a directory for your module.
    Within that, create a directory called <file>src</file>.
 
-{{% alert title="Tip" color="tip" %}}
+   {{% alert title="Tip" color="tip" %}}
 
-If you are writing your module using Python, you can use this [module generator tool](https://github.com/viam-labs/generator-viam-module) to generate stub files for the new API as well as a new module that implements the new API.
+   If you are writing your module using Python, you can use this [module generator tool](https://github.com/viam-labs/generator-viam-module) to generate stub files for the new API as well as a new {{< glossary_tooltip term_id="module" text="module" >}} that implements the new API.
 
-{{% /alert %}}
+   {{% /alert %}}
 
-4. Define your new API:
+1. Define your new API:
 
-   - [Write the proto](https://protobuf.dev/programming-guides/proto3/) methods in a `<subtype name>.proto` file inside your <file>src/proto</file> directory.
+   - [Create the proto](https://protobuf.dev/programming-guides/proto3/) methods in a `<subtype name>.proto` file inside your <file>src/proto</file> directory.
+     For reference:
      - [Example modular component proto file](https://github.com/viamrobotics/viam-python-sdk/blob/main/examples/complex_module/src/proto/gizmo.proto)
      - [Example modular service proto file](https://github.com/viam-labs/speech/blob/main/src/speech/proto/speech.proto)
-     - [Built-in Viam resource proto files for reference](https://github.com/viamrobotics/api/tree/main/proto/viam)
-   - And define the proto methods in a protobuf-supported language such as Python or Go in a file called `api.py` or `api.go`.
```

Comment:
I think this is the same https://docs.viam.com/modular-resources/create/#code-a-new-resource-model? So I've moved it to Next steps? Is that wrong?

Reply:
Seems correct to me but @JessamyT could confirm

Reply:
No I do not think that is right. Creating a model doesn't involve proto methods does it? This is the underlying API definition stuff, not the model implementation, as far as I know. That's why we called them "api.py" or whatever--to indicate that they're part of the API definition as opposed to the model definition.

Reply:
ok I've put it back